pkg/releaseutil/kind_sorter.go

```diff
@@ -138,9 +139,10 @@ func lessByKind(a interface{}, b interface{}, kindA string, kindB string, o Kind
 	second, bok := ordering[kindB]
 
 	if !aok && !bok {
-		// if both are unknown then sort alphabetically by kind, keep original order if same kind
-		if kindA != kindB {
-			return kindA < kindB
+		// if install, keep CRs installing order as the order of CRs in the raw manifests
+		// if uninstall, keep CRs uninstalling order as the reversed order of CRs in the raw manifests
```

Comment:
The ordering doesn't just apply to CRs. There are numerous resources Helm isn't explicitly tracking (e.g., Endpoint) that are treated the same. Can the comment read that Kinds Helm is not tracking, that are not typically used in charts, are sorted alpha or reverse alpha after the ones helm is tracking.

Reply:
Yes, the ordering actually applies to all unknown kinds that are not listed in `InstallOrder` or `UninstallOrder`, and I have updated the comment.

Now, the whole order rules should be:

1. install or uninstall known kinds listed in `InstallOrder` or `UninstallOrder` firstly, then unknown kinds
2. install or uninstall known kinds as the order in `InstallOrder` or `UninstallOrder`
3. install unknown kinds as the order in original manifests/hooks, uninstall unknown kinds as the *reversed* order in original manifests/hooks